app/services/conversation_bot.rb

```diff
@@ -1,13 +1,297 @@
 class ConversationBot
+  NOW_TIME = /now|ahora/i.freeze
 
+  CALL_FOR_HELP = /\Ahelp|ayuda\z/i.freeze
+
+  NUMBER_STRINGS = {
+      0 => [/0/, /zero/, /cero/],
+      1 => [/1/, /one/, /uno/],
+      2 => [/2/, /two/, /dos/],
+      3 => [/3/, /three/, /tres/],
+      4 => [/4/, /four/, /cuatro/]
+  }.freeze
+
+  DONT_KNOW_FRAGMENTS = [/don't know/i, /no se/i, /no sé/i].freeze
```

Comment:
other possibilities: /dont know/, /dunno/, /unsure/. These are texts, we should include likely typos or txt-speak like '4 u'